android/res/xml/prefs_main.xml

```diff
@@ -76,6 +76,13 @@
       android:summary="@string/enable_screen_sleep_description"
       android:defaultValue="false"
       android:order="16"/>
+
+    <SwitchPreferenceCompat
```

Comment:
Do we need this option at all? Just enable this feature by default without adding the option.

Reply:
We need it for security.

Reply:
I also think it should be configurable. This behaviour is surprising to the user. Therefore I think it should be opt-in and preference title/description is good place to let user know what will happen once enabled.